src/processors/other/LevelDetective.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "../../modules/chowdsp_utils/modules/gui/chowdsp_visualizers/CompressorPlots/chowdsp_LevelDetectorVisualizer.h"
```

Comment:
I don't think we should need this include? `<chowdsp_visualizers/chowdsp_visualizers.h>` should be included in JuceHeader.h, and which should have this header included internally.